pagerduty/komand_pagerduty/actions/get_on_call/action.py

```diff
@@ -27,16 +27,28 @@ def run(self, params={}):
         response_object = response.json()
 
         user_ids = []
-        for oncall_object in response_object.get("oncalls"):
+        for oncall_object in response_object.get("oncalls", []):
             try:
                 user_ids.append(oncall_object["user"]["id"])
-            except Exception as e:
-                self.logger.info(f"User ID not available: {str(e)}")
+            except KeyError as e:
+                self.logger.warning(f"User ID not available: {str(e)}")
                 pass
 
-        users = []
-        for user_id in user_ids:
-            user_response = self.connection.api_connection.get(f"https://api.pagerduty.com/users/{user_id}")
-            users.append(user_response.json().get("user"))
+        users = asyncio.run(self.async_get_users(user_ids))
 
         return {Output.USERS: insightconnect_plugin_runtime.helper.clean(users)}
+
+    async def async_get_users(self, user_ids: list) -> list:
```

Comment:
```suggestion
    async def async_get_users(self, user_ids: [str]) -> list:
```
Does this function accept a list of strings? What is in the list? It's best to type the list when possible.